src/RealAntennasProject/Precompute/Jobs.cs

```diff
@@ -88,7 +90,7 @@ public struct CalcAntennaBodyNoise : IJobParallelForDefer
         [WriteOnly] public NativeArray<float> bodyNoise;
         public void Execute(int index)
         {
-            bodyNoise[index] = rxHome[index] ? Precompute.NoiseFromOccluders(rxPos[index], rxGain[index], rxDir[index], rxFreq[index], rxBeamwidth[index], occluders) : rxPrecalcNoise[index];
+            bodyNoise[index] = rxTracking[index] ? Precompute.NoiseFromOccluders(rxPos[index], rxGain[index], rxDir[index], rxFreq[index], rxBeamwidth[index], occluders) : rxPrecalcNoise[index];
         }
```

Comment:
I wonder about this particular change.  The purpose of this job was to defer the noise calculation for antennas whose pointing was not determined earlier, specifically:  the ground station antennas that are allowed to "simultaneously" point at any possible peer (as a gameplay simplification).

But isTracking is also changing, from:
```
        public virtual bool CanTarget => Shape != AntennaShape.Omni && (ParentNode == null || !ParentNode.isHome);
```
 to:
```
        public bool IsTracking => Shape != AntennaShape.Omni && Target == null;
```
I don't know if those are the same, ie do only ground stations have Target == null?
But this is the only purpose of isTracking -- to indicate this non-physical idea of how a given antenna is pointing.

Reply:
To your question
> I don't know if those are the same, ie do only ground stations have Target == null?

I think the answer is yes, because this is how Target is initialized, where we see some of the logic that was in CanTarget:
```
            if (config.HasNode("TARGET"))
                Target = Targeting.AntennaTarget.LoadFromConfig(config.GetNode("TARGET"), this);
            else if (Shape != AntennaShape.Omni && (ParentNode == null || !ParentNode.isHome) && !(Target?.Validate() == true) && HighLogic.LoadedSceneHasPlanetarium)
                Target = Targeting.AntennaTarget.LoadFromConfig(SetDefaultTarget(), this);
```

The only intended change in behaviour in the changes surrounding the introduction of IsTracking is that you can explicitly configure an antenna to have this kind of track-everything-at-once behaviour using `TARGET {}`; Σκοπός uses that for its ground stations.

Those ground stations are not home, so that you cannot downlink your science to a random Орбита station (or maybe even someone’s TVRO dish!) using the entire C band.